appveyor.yml

```diff
@@ -0,0 +1,30 @@
+version: 1.0.{build}
+image: Visual Studio 2017
+environment:
+  PATH: C:\Python37-x64;%PATH%
+  PythonDir: C:\Python37-x64
+
+cache:
+- C:\Users\appveyor\AppData\Roaming\stack\
+- C:\Users\appveyor\AppData\Local\Programs\stack
+
+init:
+
+build_script:
+- python -mpip install numpy
+- curl -sS -ostack.zip -L --insecure https://get.haskellstack.org/stable/windows-i386.zip
+- 7z x stack.zip stack.exe
+- stack --arch x86_64 %APPVEYOR_BUILD_FOLDER%\scripts\Package.hs
+
+artifacts:
+- path: Dataframes-Win-x64-v141.7z
```

Comment:
Having this 7z file, is it possible to tell from which commit it was built?

Reply:
It's not possible.
What is the use-case? At the moment commit is in the name of directory where this file gets uploaded, however after download it is not clear. We can adjust filename, add some additional information inside or sth else. The remark should be easily addressable but I'd defer this until we actually need this information (then we know what is best way to expose it).